extensions/binding/org.eclipse.smarthome.binding.lirc/README.md

```diff
@@ -40,7 +40,7 @@ a bridge is configured.
 ### Things
 
 ```xtend
-Bridge lirc:bridge:local [ host="192.168.1.120", port="9001" ] {
+Bridge lirc:bridge:local [ host="192.168.1.120", portNumber="9001" ] {
```

Comment:
Looking at https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.lirc/ESH-INF/thing/bridge.xml , we can confirm that the parameter name is `portNumber` and not `port`.
But its type is integer and so it should not be necessary to surround the value with "".

Reply:
You are right, but fixing the parameter name is the relevant change here and the notation is perfectly valid.